PearlPCSup/PearlPC.proto

```diff
@@ -78,5 +101,26 @@ get_memory{
 }
 
 reset_error {
-    out "er"
+    out "er";
+	wait 100;
+}
+
+set_pos_lim {
+    out "d+%#04d";
+	wait 100;
+}
+
+set_neg_lim {
+    out "d-%#04d";
```

Comment:
I think this may be `%#01d`

Reply:
I think you're right for the `o+` and `o-` commands, I've changed that in those commands. For `d+` and `d-` I think 4 digits is correct according to the manual.